src/site/_includes/shows-index.njk

```diff
@@ -1,26 +1,36 @@
 ---
-layout: default
+layout: default-next
 ---
 {% from 'macros/shows-card.njk' import showsCard with context %}
-<div>
-  {# Make sure Actions are first in the tab order. #}
-  <div class="w-actions">
+
+<div class="shows">
+{# @TODO fix subscribe button #}
+ {# <div class="w-actions">
     {% include 'partials/subscribe-action.njk' %}
-  </div>
+  </div> #}
 
-  <div class="w-layout-container w-pb--non">
-    <header class="w-page-header">
-      <h1 class="w-page-header__headline">{{ title }}</h1>
-      <p class="w-page-header__copy">{{ description }}</p>
-    </header>
-  </div>
-  <section class="w-grid">
-    <div class="w-grid__columns w-grid__columns--three" role="list">
+  {# Hero starts #}
+  <header class="region bg-mid-bg">
+    <div class="wrapper">
+      <div class="headline all-center flow">
+        <h2 class="headline__title">{{ title }}</h2>
+        <p>{{ description }}</p>
+      </div>
+    </div>
+  </header>
+  {# Hero ends #}
+
+  <div class="region">
+    {# Shows start #}
+    <div class="wrapper auto-grid">
```

Comment:
This is rendering 4 columns on ~1500px viewport, while /blog renders only 3 columns. I think 4 is too squeezed here.